paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtxTest.kt

```diff
@@ -5,6 +5,8 @@ import android.graphics.Color
 import com.google.common.truth.Truth.assertThat
 import com.stripe.android.paymentsheet.addresselement.AddressDetails
 import org.junit.Test
+import java.security.InvalidParameterException
```

Comment:
Still probably worth making this change here.

Reply:
Made the change in the `Ktx` file but looks like `assertFailsWith` does not actually check the exception type. Will update these with a different check.

Reply:
FWIW, I don't see the change in the Ktx file either.

Reply:
I just flat out forgot to push the change lol. Pushed now.